mesh/mesh.hpp

```diff
@@ -1335,7 +1335,9 @@ class Mesh
 
    /** Return the vertex index of boundary element i. (1D)
        Return the edge index of boundary element i. (2D)
-       Return the face index of boundary element i. (3D) */
+       Return the face index of boundary element i. (3D)
+
+       This is the same as GetBdrFace(). */
    int GetBdrElementEdgeIndex(int i) const;
```

Comment:
Maybe it makes sense to deprecate this one, since it is equivalent to `GetBdrFace`?

Reply:
I'd be supportive of that. (The usage of the word "edge" in this function name is weird/confusing).

Reply:
Done in 823eef1d9